README.md

```diff
@@ -309,10 +309,16 @@ transformScale: ?number = 1,
 // dragged over.
 preventCollision: ?boolean = false;
 
-// If true, droppable elements (with "droppable" attribute) 
+// If true, droppable elements (with `draggable={true}` attribute) 
 // can be dropped on the grid. It triggers "onDrop" callback
 // with position and event object as parameters. 
 // It can be useful for dropping an element in a specific position
+//
+// NOTE: In case of using Firefox you should add
+// `onDragStart={e => e.dataTransfer.setData('text/plain', '')}` attribute
```

Comment:
Interesting. Does it actually have to be this bizarre `e.dataTransfer.setData()` bit, or can it just be an empty function?

Reply:
I agree with @STRML this looks really user-unfriendly. Is there no workaround for this? Sorry I'm on vacation and it's a bit hard to run the project atm.

Reply:
@STRML @n1ghtmare unfortunately, without that bizarre code "draggable" won't work in firefox